{{cookiecutter.project_slug}}/runtests

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+set -e
+tox "$@"
```

Comment:
Why should runtests run tox? As I mentioned when we talked earlier, I see tox as something that wraps the tests and runs them with different python versions or packages. Tox should be the one running this file imho.
Like you said yourself, it should be fast to run the tests locally, this will be slow since it tries to run all the tox tests. Also, tox most likely defines several python versions that the developer does not have installed locally, so the tests will fail due to missing Python versions.

Tox.ini should run `runtests` not the other way around and runtests should run py.tests. At least imho.